assets/js/modules/index.test.js

```diff
@@ -35,7 +35,7 @@ function directories( relativePath ) {
 
 function getComponentNames( componentPath ) {
 	return fs.readdirSync( componentPath )
-		.filter( ( name ) => ! /^index|\.test\.js$/.test( name ) )
+		.filter( ( name ) => ! /^index|\.test|\.stories\.js$/.test( name ) )
```

Comment:
Maybe a little easier to read?
```suggestion
		.filter( ( name ) => ! /^index|\.(test|stories)\.js$/.test( name ) )
```